lib/plugins.js

```diff
@@ -98,12 +96,24 @@ const loadPackages = (rootPath) => {
     });
   };
 
-  const plugins = loadDeps(json.dependencies);
-  const devPlugins = loadDeps(json.devDependencies, true);
-  const optPlugins = loadDeps(json.optionalDependencies, true);
-  const allPlugins = plugins.concat(devPlugins, optPlugins);
+  const uniqueDeps = function(/*mainDeps, ...otherDeps*/) {
+    return slice.call(arguments, 0).map(deps => {
+      return Object.keys(deps || {});
+    }).map((dep, index, deps) => {
+      if (!index) { return dep; }
+      return dep.filter((dependency) => {
+        return deps[0].indexOf(dependency) === -1;
+      });
+    });
+  };
```

Comment:
Hm, actually, to retain the `isDev` behavior, the current code kind of makes sense. A little less obvious for sure, though 😟